ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-697] Display absent capital records

## Description of change
Display empty savings, national savings certificates, assets and investments in the cya, review, completed and returned application pages if the question was asked in apply (i.e. for either way, indictable, already in crown court cases). 

Also persist national saving certificate 'none' entry in the form

## Link to relevant ticket
[CRIMAPP-697](https://dsdmoj.atlassian.net/browse/CRIMAPP-697)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

**Check your answer screen when asked for asset, savings, national saving certificate and investment info**

<img width="1032" alt="Screenshot 2024-04-09 at 14 54 03" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/dce950b5-ac97-401b-bf5d-e16ce9748172">
<img width="1032" alt="Screenshot 2024-04-09 at 14 54 08" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/ff297648-f9d1-4ebd-8ede-b1c15381e341">

**Review application screen**

<img width="1032" alt="Screenshot 2024-04-09 at 14 53 39" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/def2c6e9-8800-4eb3-b18b-9caf48907874">

**Completed application screen**

<img width="1032" alt="Screenshot 2024-04-09 at 14 52 27" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/64f769ac-8ddf-4d18-9584-c9bd8e2c9238">

**Check your answer screen when not asked for asset, savings, national saving certificate and investment info**

<img width="1032" alt="Screenshot 2024-04-09 at 14 54 38" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/91376f83-a6c8-45cd-86a6-0c867d485069">

**Persist national savings certificate value**

https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/e83270e7-05aa-46c4-b3b2-782257d98b8c

## How to manually test the feature

Start a means tested application with a case type of either way, indictable or already in crown court 
Enter a nino 
Select no benefits
Fill in the information until you reach the capital screens (starting with assets)
Select no to assets, savings, national savings certificate and investments
When you get to the cya page, confirm that you can see the assets, savings, national savings certificate and investments sections 
Confirm that the design matches the above 
Check the links take you to the 'which ___ does the client have' screen 
Then navigate to the review the application screen and confirm the same
Then submit the application and navigate to the completed application page and confirm the design is the same and no change links appear 

Secondly, start a means tested application with a case type that is **not** either way, indictable or already in crown court 
Follow the same steps as above until you reach the capital screens
You should not be asked for asset, savings, national saving certificate and investment info
Answer the questions until you reach the cya page
Confirm that the design matches the above screenshot and that the assets, savings, national savings certificate and investments sections are not visible
Then navigate to the review the application screen and confirm the same
Then submit the application and navigate to the completed application page and confirm the design is the same

[CRIMAPP-697]: https://dsdmoj.atlassian.net/browse/CRIMAPP-697?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
